src/align.rs

```diff
@@ -172,13 +178,17 @@ impl<T: Copy, C: AlignedByteChunk> AlignedVec<T, C> {
     pub fn as_slice(&self) -> &[T] {
         // SAFETY: The first `len` elements have been
         // initialized to `T`s in `Self::resize_with`.
+        // SAFETY: the pointer is sufficient aligned, as the chunks are always over-aligned with
+        // respect to `T`.
```

Comment:
Nit (typo in `sufficiently`):

```suggestion
        // SAFETY: The pointer is sufficiently aligned,
        // as the chunks are always over-aligned with respect to `T`.
```

Can you do this for the other comments, too?